arm-sql/2014-04-01/swagger/secureConnectionPolicy.json

```diff
@@ -0,0 +1,237 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/connectionPolicies/Default": {
+            "put": {
+                "tags": [
+                    "SecureConnection"
+                ],
+                "operationId": "Databases_CreateOrUpdateSecureConnectionPolicy",
```

Comment:
There are secure Database connection policies and secure Server connection policies.  They are different entities with different properties.  I would like to keep them separated and in the same operation group as the entity they effect (database/server)

---
In reply to: [80165892](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80165892) [](ancestors = 80165892)